File: codeforces/low_difficulty/difficulty_1000/7.caps_lock_[131A]/caps_lock.c
#include <stdio.h>
#include <string.h>
#include <ctype.h>
#include <stdbool.h>
/* Author: José Rodolfo (jric2002) */
int main() {
  char w[101], nw[101];
  unsigned short int size_w, i;
  bool s;
  scanf("%s", &w);
  size_w = strlen(w);
  w[size_w] = '\0';
  s = true;
  nw[0] = ((islower(w[0])) ? (char)toupper(w[0]) : (char)tolower(w[0]));
  i = 1;
  while (i < size_w) {
    if (!isupper(w[i])) {
      s = false;
      break;
    }
    nw[i] = (char)tolower(w[i]);
    i++;
  }
  nw[i] = '\0';
  if (s) {
    printf("%s\n", nw);
  }
  else {
    printf("%s\n", w);
  }
  return 0;
}

File: codeforces/low_difficulty/difficulty_1000/7.caps_lock_[131A]/caps_lock.py
# Author: José Rodolfo (jric2002)
w = str(input())
size_w = len(w)
s = True
nw = ""
nw = nw + (w[0].upper() if (w[0].islower()) else w[0].lower())
i = 1
while (i < size_w):
  if (not(w[i].isupper())):
    s = False
    break
  nw = nw + w[i].lower()
  i += 1
print(nw) if (s) else print(w)